react/store/MyReturnsPageAdd.tsx

```diff
@@ -266,6 +266,12 @@ class MyReturnsPageAdd extends Component<any, State> {
       })
     }
 
+    if (this.state.settings.hidePaymentMethodSelection) {
+      this.setState({
+        paymentMethod: 'Default',
```

Comment:
@cesarocampov @filafb @zekivtex 

All refund forms REQUIRE the payment method before creating a request.
If the setting is turned `ON` there's no input for that, so I chose to pass the value as 'Default' when creating the request.

The order payment method is not directly available:

<img width="524" alt="Payment data" src="https://user-images.githubusercontent.com/50715158/154866668-d20c1d78-16e3-4c55-923a-f96adb7bcab9.png">

We can parse the exact method chosen for the order, but it would require more time, as we should test to prove we are returning the correct values. I think we can backlog it.

Reply:
Resolved: we are passing a translatable message.
`Refund with order payment method`